tests/integration/test_phase.py

```diff
@@ -0,0 +1,227 @@
+import logging
+from time import sleep
+from unittest import TestCase
+from unittest.mock import patch, MagicMock
+
+from Constants import CURRENT_TESTNET, RewardsType, MUTEZ_PER_TEZ
+from api.provider_factory import ProviderFactory
+from calc.phased_payment_calculator import PhasedPaymentCalculator
+from calc.calculate_phaseMapping import CalculatePhaseMapping
+from calc.calculate_phaseMerge import CalculatePhaseMerge
+from calc.calculate_phaseZeroBalance import CalculatePhaseZeroBalance
+from calc.service_fee_calculator import ServiceFeeCalculator
+from config.yaml_baking_conf_parser import BakingYamlConfParser
+from exception.api_provider import ApiProviderException
+from model.rules_model import RulesModel
+from model.baking_conf import BakingConf
+from NetworkConfiguration import default_network_config_map
+from tests.utils import mock_request_get
+
+PAYOUT_CYCLE = 500
+
+logger = logging.getLogger("unittesting")
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler())
+
+
+class TestCalculatePhases(TestCase):
+
+    baking_config = (
```

Comment:
Could we use the make_config util to make this?

Reply:
done